backend/DevUp/DevUp.Infrastructure.Postgres/JwtIdentity/Validators/UsernameValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DevUp.Infrastructure.Postgres.JwtIdentity.Dtos;
+using Microsoft.AspNetCore.Identity;
+
+namespace DevUp.Infrastructure.Postgres.JwtIdentity.Validators
+{
+    internal class UsernameValidator : IUserValidator<UserDto>
+    {
+        public async Task<IdentityResult> ValidateAsync(UserManager<UserDto> manager, UserDto user)
```

Comment:
Could be replaced with existing solution => https://docs.fluentvalidation.net/

Reply:
Good suggestion. Will be handled as part of #28. Let's ignore it on this one for now.